components/reusable-components/pagination/HomePagination.tsx

```diff
@@ -1,30 +1,30 @@
 import React, { useEffect, useState } from 'react';
 import { fetchItems, Item } from './lib/data';
-import StructurePagination from './StructurePagination';
 import Dropdown from './Dropdown';
 import styles from './HomePagination.module.scss';
 
+/* eslint no-unused-vars: "off" */
 const HomePagination: React.FC = () => {
   const [items, setItems] = useState<Item[]>([]);
   const [currentPage, setCurrentPage] = useState(1);
-  const [totalPages, setTotalPages] = useState(1);
-  const [itemsPerPage, setItemsPerPage] = useState(25);
+  const [TOTAL_PAGES, SET_TOTAL_PAGES] = useState(1);
+  const [ITEMS_PER_PAGE, SET_ITEMS_PER_PAGE] = useState(25);
```

Comment:
naming const with capital letters doesnt apply to useState, please revert this changes